src/lib.rs

```diff
@@ -102,6 +111,26 @@ mod tests {
 
         update_table("test", &b).unwrap();
         assert_eq!(ex_2, read_table("test".to_string()));
+
+        drop_table("test");
+    }
+
+    #[test]
+    #[allow(unused_must_use)]
+    // This is not a benchmark - it is just to make sure this can be done correctly
+    fn it_can_create_100_tables_and_drop_them_all() {
+        for n in 1..101 {
+            let table = fmt::format(format_args!("{}", n));
```

Comment:
This is equivalent to `let table = format!("{}", n);`

It's vary rare to need `format_args!` unless you're writing your own macros.